extensions/amp-story-player/validator-amp-story-player.protoascii

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-story-player
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-story-player"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-story-player>
+  html_format: AMP
+  tag_name: "AMP-STORY-PLAYER"
+  requires_extension: "amp-story-player"
+  child_tags: {
+    child_tag_name_oneof: "A"
+  }
```

Comment:
To double check: no mandatory `poster` attribute then?

Reply:
The poster is a CSS custom property and not an attribute 😟

Reply:
I was thinking of adding a regex but IIUC we would have to apply it in `validator-main.protoascii`, which contains the validation for the `<a>` tag. Not sure if there's a way to validate this for the `amp-story-player` context only. 

/cc @Gregable here is the spec we are trying to validate, specifically the `--story-player-poster` CSS variable inside the `style` attribute of the `<a>` inside the `<amp-story-player>`. Is this possible for us to validate with the current tools?

```
<amp-story-player>
        <a href="https://exampel.com/story1.html"
            style="--story-player-poster: url('https://example1.com/img/promo3x4.jpg');">
        </a>
</amp-story-player>
```

Reply:
/cc @ampproject/wg-caching

Reply:
Discussed solutions over chat. Approving this CL for now though.